rekognition-apis/1-object-detection.ipynb

```diff
@@ -150,7 +169,10 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "scrolled": true,
```

Comment:
What does this line do?

Reply:
I found some info in a [thread](https://github.com/jupyterlab/jupyterlab/issues/4028) - I guess it's mostly styling related for getting some long output automatically scrolled to bottom - some arguing it should be the default. I guess I don't think I have a strong opinion.

Reply:
Perfect, thanks. I looked for docs on the attribute but couldn't find anything.